src/stylesheets/App.css

```diff
@@ -0,0 +1,354 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700&family=Roboto+Slab:wght@100;200;300;400;500;600;700&display=swap');
+
+:root {
+  --azure: #0290ff;
+  --gray: #e8e8e8;
+  --black: #121212;
+  --blue: #4386bf;
+  --medium-blue: #307bbe;
+  --dodger-blue: #379cf6;
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #fafafa;
+  font-family: 'Montserrat', sans-serif;
+  max-width: 1440px;
+  margin: 0 auto;
+}
+
+li {
+  list-style: none;
+}
+
+.flex {
+  display: flex;
+}
+
+.p-100 {
+  padding: 0 100px;
+}
+
+.align-c {
+  align-items: center;
+}
+
+.just-c {
+  justify-content: center;
+}
+
+.just-sb {
+  justify-content: space-between;
+}
+
+.pos-rel {
+  position: relative;
+}
+
+.pointer {
+  cursor: pointer;
+}
+
+.op-5 {
+  opacity: 0.5;
+}
+
+.fw-300 {
+  font-weight: 300;
+}
+
+.nav {
```

Comment:
- [ ] It would be better if we implement the same design as the previous code reviewers mentioned. Let us take a close look at our `nav`, we can see from the design, there are 4 elements but in our design, we have 5 elements. Let us delete the extra element while implementing the same functionality